js/lesson2/tutorial.md

```diff
@@ -205,6 +208,18 @@ if (totalPeople > pizzas) {
 }
 ```
 
+#### Ternaries - Additional learning
+
+In the latest versions of JavaScript you may see if/else statements written slighlty differently. We believe teaching the above way first helps cement how they work, however if you're working with a code base that uses the latest JavaScript you will see it written this way.
```

Comment:
@KimberleyCook not sure if I'd frame this in the context of older JavaScript vs newer JavaScript. Ternary statements have been around for quite a while. Maybe explain that they are a different way to write if/else statements that return a value.